### DISCUSSION TITLE
Laragon server doesn't work when offline

### DISCUSSION THREAD

**Discussion by Martian4x** [leokhoa/laragon#814] [Q&A] [answered]
I noticed that whenever I disconnect the PC from the internet the Laragon apache server also stops working. The Laravel app acts as if it is hosted online while everything is hosted locally. But when I enable the wi-fi everything works perfectly. Is there a way to disable this?

I'm working on windows 11, Laragon full 6.0 220916 php-8.2.17-nts

**Comment by ppfoong** [comment, marked as answer]
You need TCP/IP up and running. 

Go to cmd and type "ipconfig", it will show your TCP/IP status.

One way to have Windows TCP/IP running without connecting to any network is to use Microsoft Loopback Adapter. You can refer to this article on how to install and enable it:
https://answers.microsoft.com/en-us/windows/forum/all/microsoft-loopback-adapter/5f643e76-5808-40d1-b468-fee3c96536bf
